packages/central-server/src/apiV2/index.js

```diff
@@ -270,6 +270,7 @@ apiV2.put('/mapOverlayGroups/:recordId', useRouteHandler(EditMapOverlayGroups));
 apiV2.put('/mapOverlayGroupRelations/:recordId', useRouteHandler(EditMapOverlayGroupRelations));
 apiV2.put('/indicators/:recordId', useRouteHandler(BESAdminEditHandler));
 apiV2.put('/projects/:recordId', useRouteHandler(BESAdminEditHandler));
+apiV2.put('/entities/:recordId', useRouteHandler(BESAdminEditHandler));
```

Comment:
Would be good to lock this down on the back end to ensure code doesn't get changed 🙏